### PR TITLE
GCP BigQuery Solution - Initial commit of solution

### PR DESCRIPTION
**Pull request by rahul0216**
Change(s):
   - Added ARM template for logic app custom connector
   - Added ARM templates for Playbooks
   - Added readme.md files
   - Added screenshots of playbook

   Reason for Change(s):
   - Initial commit

   Version Updated:
   - Initial Version 1.0.0

   Testing Completed:
   - End to end deployment testing
   - End to end functional testing

### CODE REVIEW

File: Solutions/Google Cloud Platform BigQuery/Playbooks/CustomConnector/GCPBigQueryCustomConnector/readme.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+<img src="../../google_logo.svg" alt="Google" style="width:150px; height:150px"/><br>
+
+This custom connector connects to Google Cloud Platform (GCP) BigQuery Service API to execute actions supported by API returns response in JSON format.
+
+### Authentication methods this connector supports
+
+*  OAuth2.0 Authentication
+
+### Prerequisites to deploy Custom Connector 
+1. Get/Verify Shodan API Endpoint Url (Default: https://bigquery.googleapis.com)
```
Comment (manishkumar1991) [inline Solutions/Google Cloud Platform BigQuery/Playbooks/CustomConnector/GCPBigQueryCustomConnector/readme.md:12]: Please change Shodan to respective keyword

**Comment by manishkumar1991** [comment]
I think logo should be provided in https://github.com/Azure/Azure-Sentinel/tree/master/Logos  , Not in solution

**Review by manishkumar1991 — APPROVED** [review]
approved

**Comment by v-sabiraj** [comment]
@rahul0216/ @manishkumar1991, please suggest if we can merge this.

**Comment by v-dvedak** [comment]
As confirmed with Manish, merging the PR

**Pull request merged**